pkg/userGRPC/controller/controller.go

```diff
@@ -0,0 +1,130 @@
+package usergrpccontroller
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/uuid"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/emptypb"
+
+	customerrors "github.com/stasBigunenko/monorepa/errors"
+	"github.com/stasBigunenko/monorepa/model"
+	pb "github.com/stasBigunenko/monorepa/pkg/userGRPC/proto"
+)
+
+type UserGRPCСontroller struct {
+	client pb.UserGRPCServiceClient
+}
+
+func New(cli pb.UserGRPCServiceClient) UserGRPCСontroller {
+	return UserGRPCСontroller{
+		client: cli,
+	}
+}
+
+func (s UserGRPCСontroller) formatError(err error, message string) error {
```

Comment:
Too many parts with `formatError` and handle _status code_. Will be better to create service with errors msg???

Reply:
You can make a new ticket for that, is a low priority one